sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/xml/XmlSuite.scala

```diff
@@ -1729,9 +1729,15 @@ class XmlSuite extends QueryTest with SharedSparkSession {
     val TAG_NAME = "tag"
     val VALUETAG_NAME = "_VALUE"
     val schema = buildSchema(
+      field(VALUETAG_NAME),
```

Comment:
Why the fields were rearranged?

Reply:
We sort the field name in ascending order. The `_VALUE` comes before `_attr`